rosette/src/Export.cc

```diff
@@ -159,10 +243,10 @@ std::map<ExportObjectKey, std::pair<ExportObjectHandler, ExportObjectType> >  ha
     {"TblObject",           {defaultObjectHandler,   ObjectCodePB::OT_TblObject} },
     {"Template",            {defaultObjectHandler,   ObjectCodePB::OT_Template} },
     {"Tuple",               {tupleObjectHandler,     ObjectCodePB::OT_Tuple} },
-    {"TupleExpr",           {defaultObjectHandler,   ObjectCodePB::OT_TupleExpr} },
+    {"TupleExpr",           {tupleExprObjectHandler, ObjectCodePB::OT_TupleExpr} },
 };
 
-uint64_t createObjectId(Code * code) {
+IdType createObjectId(Code * code) {
     // Rosette internally, uses the object pointer but for export, this results in
```

Comment:
Punctuation: Rosette internally uses the object pointer but for export. This

Reply:
Done

Reply:
Oops! I misparsed it.  I think it should say, "We can't just use the pointer to the object as a unique id when exporting, because when an object goes out of scope its memory is reused.  Therefore we create a unique id for each object as it's constructed."

Reply:
// Rosette internally uses the object pointer. We can't just use the pointer
    // to the object as a unique id when exporting, because when an object goes
    // out of scope its memory is reused. Therefore we create a unique id for each
    // object as it's constructed.


This will be in the next PR.